yb-voyager/src/query/queryissue/issues_dml.go

```diff
@@ -56,3 +56,16 @@ var xmlFunctionsIssue = issue.Issue{
 func NewXmlFunctionsIssue(objectType string, objectName string, sqlStatement string) QueryIssue {
 	return newQueryIssue(xmlFunctionsIssue, objectType, objectName, sqlStatement, map[string]interface{}{})
 }
+
+var loFunctionsIssue = issue.Issue{
```

Comment:
Write a test case in issues_dml_test(create this) to assert that it fails when executed on YB.